OpenFHEConfig.cmake.in

```diff
@@ -40,20 +40,13 @@ set(OpenFHE_CXX_COMPILER_VERSION "@CMAKE_CXX_COMPILER_VERSION@")
 set(OpenFHE_STATIC "@BUILD_STATIC@")
 set(OpenFHE_SHARED "@BUILD_SHARED@")
 set(OpenFHE_TCM "@WITH_TCM@")
-set(OpenFHE_WITH_INTEL_HEXL "@WITH_INTEL_HEXL@")
 set(OpenFHE_OPENMP "@WITH_OPENMP@")
 set(OpenFHE_NATIVE_SIZE "@NATIVE_SIZE@")
 set(OpenFHE_CKKS_M_FACTOR "@CKKS_M_FACTOR@")
 set(OpenFHE_NATIVEOPT "@WITH_NATIVEOPT@")
 
 # Math Backend
-if("@WITH_BE2@")
-	set(OpenFHE_BACKEND "BE2")
-elseif("@WITH_BE4@")
-	set(OpenFHE_BACKEND "BE4")
-elseif("@WITH_NTL@")
-	set(OpenFHE_BACKEND "NTL")
-endif()
+set(OpenFHE_BACKEND "@MATHBACKEND@")
```

Comment:
Although the previous logic could potentially set OpenFHE_BACKEND to an incorrect backend, the values were 'BE2', 'BE4' or 'NTL'.
The new logic will set OpenFHE_BACKEND to the correct backend, but the values are '2', '4' or '6'.
Assume this could break something downstream that is looking for the specific values 'BE2', 'BE4' or 'NTL'?

Reply:
OpenFHE_BACKEND is not used anywhere so far. I have checked multiple repos for that, such as OpenFHE, Dualisade, Secureplus-Crypto, Secureplus-CFW, OpenFHE-WASM, openfhe-genomic-examples, openfhe-integer-examples and openfhe-logreg-training-examples

Reply:
Can confirm, we don't change it or check it to my knowledge